packages/app/components/constants.ts

```diff
@@ -1,6 +1,8 @@
 import { AbiType, StateMutabilityType, AbiItem } from "web3-utils";
 import {
+  // KOVAN_STAKING_ADDRESS,
   KOVAN_PASSPORT_FACTORY_ADDRESS,
+  // ROPSTEN_STAKING_ADDRESS,
   ROPSTEN_PASSPORT_FACTORY_ADDRESS,
 } from "@cabindao/nft-passport-contracts/artifacts/addresses";
```

Comment:
I deployed the staking contract already to npm, kovan, ropsten just to get the front end development connected and not throwing errors, I expect the contracts themselves will have more iteration